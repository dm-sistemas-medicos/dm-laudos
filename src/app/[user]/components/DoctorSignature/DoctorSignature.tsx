import Image from "next/image";

import users from "@/users";

import styles from "./DoctorSignature.module.scss";

const DoctorSignature = () => {
  const userInfo = users["mariana-machoski"];

  return (
    <div className={styles["doctor-signature"]}>
      <Image src={"/assinatura.png"} width={75} height={75} alt="Assinatura" />
      {userInfo.signature.map((text, i) => (
        <p key={i}>{text}</p>
      ))}
    </div>
  );
};

export default DoctorSignature;
