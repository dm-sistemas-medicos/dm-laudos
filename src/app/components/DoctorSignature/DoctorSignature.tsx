import Image from "next/image";

import userInfo from "@/users/mariana-machoski/user-info.json";

import styles from "./DoctorSignature.module.scss";

const DoctorSignature = () => (
  <div
    className={styles["doctor-signature"]}
    data-testid="doctor-signature-container"
  >
    <Image src={"/assinatura.png"} width={75} height={75} alt="Assinatura" />
    {userInfo.signature.map((text, i) => (
      <p key={i}>{text}</p>
    ))}
  </div>
);

export default DoctorSignature;
