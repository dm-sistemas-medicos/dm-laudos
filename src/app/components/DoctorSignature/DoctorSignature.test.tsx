import "@testing-library/jest-dom";

import { render } from "@testing-library/react";

import DoctorSignature from "./DoctorSignature";

describe("DoctorSignature component", () => {
  it("renders the component", () => {
    const { container } = render(<DoctorSignature />);

    expect(container).toMatchSnapshot();
  });
});
