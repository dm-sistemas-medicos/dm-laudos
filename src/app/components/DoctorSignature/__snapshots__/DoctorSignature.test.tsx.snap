// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoctorSignature component renders the component 1`] = `
<div>
  <div
    class="doctor-signature"
  >
    <img
      alt="Assinatura"
      data-nimg="1"
      decoding="async"
      height="75"
      loading="lazy"
      src="/_next/image?url=%2Fassinatura.png&w=256&q=75"
      srcset="/_next/image?url=%2Fassinatura.png&w=96&q=75 1x, /_next/image?url=%2Fassinatura.png&w=256&q=75 2x"
      style="color: transparent;"
      width="75"
    />
    <p>
      Dra. Mariana Carolina C. Machoski
    </p>
    <p>
      MÃ©dica Otorrinolaringologista
    </p>
    <p>
      CRM-SC 35.098 | RQE 23.991
    </p>
  </div>
</div>
`;
